PnX-SI/GeoNature
# [IMPORT] test(front): add fixture and test for list import

## [Lien tâche lié dans le projet Import V3](https://github.com/orgs/PnX-SI/projects/13/views/7?pane=issue&itemId=54889918)

- Add test for number of rows
- Test for columns displayed (config)
- Test for filter feature --> :warning:  need to think about these tests ( only work if you have import valid_file from synthese and occhab in folder of test backend) (different strategy possible with pro and cons --> https://learn.cypress.io/advanced-cypress-concepts/database-initialization-and-seeding ) 
  * Search term exist in request
  * Content of cell column exist
  * Number of rows expected based on search term

- Add devices size in order to test for different devices

Screenshot for test frontend : 

![test_frontend_search_import_list](https://github.com/PnX-SI/GeoNature/assets/111564663/df226ba5-eb47-4ffc-a4f7-4408d68b044c)


Reviewed-by: andriacap
